inlang/source-code/cli/src/commands/module/build/buildCommandAction.test.ts

```diff
@@ -0,0 +1,42 @@
+import { it, expect } from 'vitest';
+import { buildCommandAction } from './command.js';
+import type { BuildActionArgs } from './command.js';
+
+// List all Node.js APIs that should be forbidden
+const forbiddenNodeImports: string[] = [
+  'fs',
+  'path',
+  'os',
+  'net',
+];
+
+it('Forbidden Node.js API Detection', async () => {
+  let originalConsoleError: (...data: any[]) => void;
+  const errorLogs: string[] = [];
+
+  // Store original console.error and intercept its output
+  originalConsoleError = console.error;
+  console.error = (...args: any[]) => {
+    errorLogs.push(args.join(' '));
+  };
+
+  for (const forbiddenImport of forbiddenNodeImports) {
+    await it(`Should detect forbidden Node.js import: ${forbiddenImport}`, async () => {
+      const args: BuildActionArgs = {
+        entry: 'testEntry.js', // Replace with an actual entry file path
+        outdir: './dist',
+        watch: false,
+      };
+
```

Comment:
How does this test work? 

I would expect multiple test files that import node APIs. I see no `testEntry.js` file however.